flink-streaming-java/src/main/java/org/apache/flink/streaming/api/checkpoint/CheckpointedFunction.java

```diff
@@ -80,7 +80,7 @@
  *         countPerKey = context.getKeyedStateStore().getReducingState(
  *                 new ReducingStateDescriptor<>("perKeyCount", new AddFunction<>(), Long.class));
  *
- *         // get the state data structure for the per-key state
+ *         // get the state data structure for the per-partition state
```

Comment:
I think it would be better to use per-operator state here. Operator is a better established term in flink.

Reply:
Agree @dawidwys , But I think if we want to modify this comment, it is best to modify it along with the variable name `countPerPartition ` and the state descriptor name `perKeyCount`.